src/test/unit/mocks/mockRequest.ts

```diff
@@ -26,3 +26,29 @@ export const mockRequest = ({
   } as any;
   return req;
 };
+
+export const acasMultipleMockRequest = ({
```

Comment:
Why do you need a new mochRequest? Can you not just provide data into the existing one?

Reply:
Removed the extra mock request and amended the controller test file to include the radio buttons inside userCase constant.